### PR TITLE
zh-translation/content/zh/blog/2021/zero-config-istio/index.md

### PR DESCRIPTION
**Pull request by my-git9**
translation-zh/content/zh/blog/2021/zero-config-istio/index.md

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
If you elect Details after the lint test, and look at the build log, you will see what is being flagged:
```
>> 776 files are free from spelling errors
    ./zh/blog/2021/zero-config-istio/index.md
       11 | publishdate: 2021-02-25 
       88 | 认设置之外，Istio 还提供了 [各种流量管理设置] (/zh/docs/concepts/traffic-managem 

>> 2 spelling errors found in 674 files
To learn how to address spelling errors, please see https://istio.io/about/contribute/build/#test-your-changes
./zh/blog/2021/zero-config-istio/index.md:41: MD009 Trailing spaces
./zh/blog/2021/zero-config-istio/index.md:34: MD012 Multiple consecutive blank lines
./zh/blog/2021/zero-config-istio/index.md:46: MD012 Multiple consecutive blank lines

```

File: content/zh/blog/2021/zero-config-istio/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Hostname=echo-cb96f8d94-rpfqz
+Hostname=echo-cb96f8d94-cgmxr
+Hostname=echo-cb96f8d94-wf4xk
+Hostname=echo-cb96f8d94-rpfqz
+Hostname=echo-cb96f8d94-cgmxr
+{{< /text >}}
+
+在这里，我们可以看到我们的请求是 [轮询（round-robin）](/zh/docs/concepts/traffic-management#load-balancing-options) 在所有后端之间进行负载平衡的。
+
+除了这些更好的默认设置之外，Istio 还提供了 [各种流量管理设置（variety of traffic management settings）](/zh/docs/concepts/traffic-management/) 的自定义，包括超时、重试等等。
```
Comment (zirain) [inline content/zh/blog/2021/zero-config-istio/index.md:69]: 除了这些默认设置之外，Istio 还提供了 [各种流量管理设置（variety of traffic management settings）](/zh/docs/concepts/traffic-management/) 的自定义设置，包括超时、重试等等。

**Review by zirain — COMMENTED** [review]
lgtm
leave to the doc maitainer

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**